dvl-a50/dvl.py

```diff
@@ -365,7 +378,8 @@ def handle_velocity(self, data: Dict[str, Any]) -> None:
             return
 
         if self.rangefinder and alt > 0.05:
-            self.mav.send_rangefinder(alt)
+            if confidence >= self.confidence_cutoff:
+                self.mav.send_rangefinder(alt, confidence)
```

Comment:
Could this not be another `and` in the line before, and drop an indentation level?